tools/msvs/msi/product.wxs

```diff
@@ -19,10 +19,14 @@
            UpgradeCode="47c07a3a-42ef-4213-a85d-8f5a59077c28">
 
     <Package Languages="!(loc.LocaleId)"
-             InstallerVersion="200"
+             InstallerVersion="500"
              Compressed="yes"
              InstallScope="perMachine"/>
 
+    <Condition Message="This application is only supported on Windows 7, Windows Server 2008 R2, or higher.">
+      <![CDATA[Installed OR (VersionNT >= 601)]]>
```

Comment:
The idea here is that this is placed in a condition table in the MSI. The installer will only launch if the conditions are all true.

`Installed` is true when uninstalling and false when installing. @orangemocha we must make sure this works as intended when upgrading, in both Vista and 7.